pyproject.toml

```
@@ -20,3 +20,7 @@ norecursedirs = [ ".*", "build", "dist", "*.egg", "data", "__pycache__", ".githu
 profile = "black"
 known_first_party = ["nf_core"]
 multi_line_output = 3
+
+[mypy]
+ignore_missing_imports = true
+follow_imports = "skip"
```

I want to see if allowing redefinitions helps with the repeated command names in the CLI interface.

```suggestion
follow_imports = "skip"
allow_redefinition = true
```